docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -0,0 +1,103 @@
+
+### Google service accounts
+
+Create a Google service account that has a private key and the **Storage Admin** role permissions. Follow these steps:
+
+1. Run the `export {VARIABLE}={value}` command to set up the following environment variables, where:
+
+    - **SA_NAME** is the name of the service account.
+    - **SA_DISPLAY_NAME** is the display name of the service account.
+    - **PROJECT** is the GCP project ID.
+    - **SECRET_FILE** is the path to the private key.
+    - **ROLE** is the **Storage Admin** role bound to the service account.
+
+    Example:
+    ```
+    export SA_NAME=my-service-account
+    export SA_DISPLAY_NAME=service-account
+    export PROJECT=service-account-012345
+    export SECRET_FILE=my-private-key-path
+    export ROLE=roles/storage.admin
+    ```
+2. When you communicate with Google Cloud for the first time, set context to your Google Cloud project. Run this command:
+    ```bash
+    gcloud config set project $PROJECT
+    ```
+3. Create a service account. Run:
+    ```bash
+    gcloud iam service-accounts create $SA_NAME --display-name $SA_DISPLAY_NAME
+    ```
+4. Add a policy binding for the **Storage Admin** role to the service account. Run:
+    ```bash
+    gcloud projects add-iam-policy-binding $PROJECT --member=serviceAccount:$SA_NAME@$PROJECT.iam.gserviceaccount.com --role=$ROLE
+    ```
+5. Create a private key for the service account:
+    ```bash
+    gcloud iam service-accounts keys create $SECRET_FILE --iam-account=$SA_NAME@$PROJECT.iam.gserviceaccount.com
+    ```
+5. Export the private key as the environment variable:
+    ```bash
+    export GCS_KEY_JSON=$(< "${SECRET_FILE}" base64 | tr -d '\n')
+    ```
+
+### ConfigMap
+
+Apply the following ConfigMap with an override onto a cluster or Minikube. Run:
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+    name: asset-store-overrides
+    namespace: kyma-installer
+    labels:
+    installer: overrides
+    component: assetstore
+    kyma-project.io/installation: ""
+type: Opaque
+data:
+    minio.gcsgateway.gcsKeyJson: "${GCS_KEY_JSON}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: asset-store-overrides
+    namespace: kyma-installer
+    labels:
+    installer: overrides
+    component: assetstore
+    kyma-project.io/installation: ""
+data:
+    minio.persistence.enabled: "false"
+    minio.gcsgateway.enabled: "true"
+    minio.defaultBucket.enabled: "false"
+    minio.gcsgateway.projectId: "${PROJECT}"
+    global.minioExternalEndpoint: "https://storage.googleapis.com"
+EOF
+```
+>**CAUTION** When you install Kyma locally from sources, you need to manually add the ConfigMap and the Secret to the `installer-config-local.yaml.tpl` template located under the `installation/resources` subfolder before you run the installation script.
+
+### Kyma installation
+
+Trigger Kyma installation or update by labeling the `kyma-installtion` custom resource. Run:
```

Comment:
Actually, the CR name is Installation -> I know that it is written similarly (`kyma-installation` custom resource) in the [local installation doc](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/04-02-local-installation.md) (point 8) but I think it is not correct, @tomekpapiernik, am I right?
As for the CR formatting, we use:
- `installations.installer.kyma-project.io` CustomResourceDefinition (CRD) 
- Installation custom resource

See the [Installation CR](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/05-02-custom-component-installation.md) for reference.

Reply:
I will change to the Installation custom resource.